recipes/dustgoggles/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "dustgoggles" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dustgoggles-{{ version }}.tar.gz
+  sha256: f86a61f507801f04793ddba405b40716fa474ff8399c48c11159475abfc9e140
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.9
+  run:
+    - cytoolz
+    - python >=3.9
+
+test:
+  imports:
+    - dustgoggles
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/millionconcepts/dustgoggles.git
+  summary: dustgoggles is a utility library for the `marslab` ecosystem. It contains a variety of generic utilities, text-parsing tools, a framework for domain-specific functional languages, and an in-memory cache framework.
```

Comment:
This is too big for a summary. Can you use the text from the GitHub repo summary? BTW, you can still keep this one are a description if you want to.

Reply:
definitely. changed in 8e0590f